### DISCUSSION TITLE
Docker | Tun2Socks

### DISCUSSION THREAD

**Discussion by af0rte** [engageub/InternetIncome#118] [General]
hi, 
i need help to understand the issue\problem i do with containers.
I have two server,  one them  have a 1 have a conection 1 GBit/s-Port
 and the other have a connection 10 Gbit. The both servers have 25 proxies and the end of day the have same trafic on earnapp (example : first have 1,4GB and the second have 1,5GB) .
 
1 server
![image](https://github.com/engageub/InternetIncome/assets/36008040/df389fc0-be34-40b7-be29-5b2e87d26e2f)

2 server 
![image](https://github.com/engageub/InternetIncome/assets/36008040/2d4e582f-b10a-46c2-b217-f08614806e5f)

how can we see in the two images, the first dont lose to much messages/packets (Block I/O), but he second lose GB of messages, why dis this happen?

thanks!

**Comment by af0rte** [comment]
This is the graphic of second server.
![image](https://github.com/engageub/InternetIncome/assets/36008040/82e1fd1f-ab06-48da-8dd8-396837b5abac)

**Comment by engageub** [comment]
Hello, 
Block I/O and Net I/O are different.  
Block I/O is the amount of data the container has written to and read from block devices on the host.
Net I/O is the amount of data the container has received and sent over its network interface. 

If you compare Net I/O,  both are same for earnapp and tun2socks. 

The traffic depends on the IP and location so do the Block I/O and Net I/O differ.

Thank you
